packages/tabs/src/TabTitle.tsx

```diff
@@ -132,17 +150,23 @@ const TabTitle: ExtendableBox<BaseTabProps, 'button'> = ({
   ariaControl,
   index,
   darkMode,
+  containerNode,
```

Comment:
if this is only used to check whether one of the tabs is focused then we can just pass in something like an `isAnyTabFocused` prop instead.

Reply:
👍